### ISSUE TITLE
only getting 2.2v to wemos

### ISSUE THREAD

**Issue by Nexetion** [sidoh/motor_driver_pcb#3]
I'm only getting 2.2v to the wemos d1 mini but the thing is pretty hot and it won't power on for some reason? Can't access the webgui

**Comment by sidoh** [comment]
The buck converter I'm using here has an adjustable output voltage. If you haven't already, try turning the screw on the top of the variable resistor until output voltage is 3.3v.

**Comment by Nexetion** [comment]
> The buck converter I'm using here has an adjustable output voltage. If you haven't already, try turning the screw on the top of the variable resistor until output voltage is 3.3v.

Yep it's at 3.3v, I'm using the same buck converter you are I'm pretty sure

**Comment by Nexetion** [comment]
I'm using your PCB

**Comment by sidoh** [comment]
Where are you seeing 2.2v?

**Comment by Nexetion** [comment]
5v pin on the wemos

**Comment by Nexetion** [comment]
> Where are you seeing 2.2v?

Got it working, used another d1 guess that other one was bad, but it seems like the stepper motor is pausing or something for like a millisecond during opening and closing any idea what that could be?
